JitsiConnection.js

```diff
@@ -25,23 +25,28 @@ export default function JitsiConnection(appID, token, options) {
 
     this.addEventListener(JitsiConnectionEvents.CONNECTION_FAILED,
         (errType, msg) => {
-            Statistics.sendEventToAll(
-                `${CONNECTION_FAILED_}.${errType}`,
-                { label: msg });
+            Statistics.sendAnalyticsAndLog(
+                createConnectionFailedEvent(errType, msg));
         });
 
     this.addEventListener(JitsiConnectionEvents.CONNECTION_DISCONNECTED,
         msg => {
             // we can see disconnects from normal tab closing of the browser
             // and then there are no msgs, but we want to log only disconnects
             // when there is real error
+            // XXX Do we need the difference in handling between the log and
+            // analytics event here?
```

Comment:
yeah it seems like this work would be when we get this questions answered, are we going to be able to clean this up?